lib/reverse_sentence.rb

```diff
@@ -1,6 +1,98 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) where n is the length of the string.
+#                                 (more line O(3n)
+# Space complexity: O(1) - in place
+
 def reverse_sentence(my_sentence)
```

Comment:
Outstanding submission.  Perfect time and space complexity.  Well done.  Do look at my suggestion regarding a helper method.